step.yml

```diff
@@ -27,6 +27,14 @@ inputs:
     - "true"
     - "false"
 
+- artifact_sources: ""
+  opts:
+    title: Artifact source
+    summary: The list fo the workflows/stages, thawhicht could generate artifacts.
+    description: 
+      'A coma separated list of workflows or stage names, which can generate artifacts. Example 1: test-apk-generator-workflow, ipa-generator-for-testers'
+    is_required: true
```

Comment:
We could also make it optional. If it is not given, all artifacts from finished workflows could be downloaded. What do you think?